src/test/java/Homework19.java

```diff
@@ -0,0 +1,88 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.TimeoutException;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        // Performing actions: login, search song, select 1st result, add to playlist
+        // LOGIN
+        login("giovanna.silva@testpro.io", "ShakaMaya1302!");
+        Thread.sleep(2000);
+
+        // Click to view playlist
+        deletePlaylistAction();
+
+        // Delete the playlist
+        confirmDelete();
+
+        // Confirm Deletion Success
+        confirmDeletionSuccess();
+
+        // Quitting the WebDriver session
+        driver.quit();
```

Comment:
**Use in test methods, the following logical split   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**